config/sync/registration_validation.settings.yml

```diff
@@ -0,0 +1,7 @@
+domains: ''
+usernames: "admin\r\nroot\r\nwebmaster\r\nwebadmin\r\npostmaster\r\nadministration\r\nadministrator"
+domains_message: 'The e-mail domain %domain has been blacklisted from registering. Please enter a different e-mail address.'
+usernames_message: 'The username contains %username, which has been blacklisted from being used when registering. Please enter a different username.'
+validation_counter: 2
+_core:
+  default_config_hash: bXfy4ebH_6Zsl-N1wtivC9-L2Js37ddprpeeDVxBVdI
```

Comment:
Did you test this?

Reply:
I did test it and it works, except the error message doesn't appear. I'm looking into it but I'm having no luck.

Reply:
So that means this pull request is not ready for tech review then?

Reply:
It is ready for tech review now.